applications/MeshMovingApplication/custom_utilities/move_mesh_utilities.cpp

```diff
@@ -54,7 +54,7 @@ void MoveMesh(const ModelPart::NodesContainerType& rNodes) {
     for (int i=0; i<num_nodes; i++) {
         const auto it_node  = nodes_begin + i;
         noalias(it_node->Coordinates()) = it_node->GetInitialPosition()
-            + it_node->FastGetSolutionStepValue(MESH_DISPLACEMENT);
+            + it_node->FastGetSolutionStepValue(MESH_DISPLACEMENT) + it_node->FastGetSolutionStepValue(DISPLACEMENT);
```

Comment:
```suggestion
            + it_node->FastGetSolutionStepValue(MESH_DISPLACEMENT) + it_node->GetValue(AUX_MESH_DISPLACEMENT);
```

Reply:
Would this work for you?

In principle yes, but this would propagate quite a bit througout the code.
Hence I would prefer to set it to a specific variable (I mean, also other variables are specific, we cannot just use what we want)

Reply:
Yes that works I believe. 

I did not understand by "other variables are specific" ? Do you mean the MESH_VELOCITY ?

Reply:
With other variables I mean things like VELOCITY/PRESSURE for fluid or DISPLACEMENT/ROTATION in structural

Those are fixed